components/03-organisms/galleries/media-grid/_yds-media-grid.scss

```diff
@@ -0,0 +1,59 @@
+@use '../../../00-tokens/tokens';
+@use '../../grid-mixins' as grid;
+
+.media-grid {
+  @include tokens.spacing-page-section;
+}
+
+.media-grid__heading {
+  @include tokens.h2-yale-new;
+
+  margin-bottom: var(--spacing-page-inner);
+}
+
+.media-grid__items {
+  @include grid.secondary(true);
+}
+
+[data-media-grid-variation='interactive'] {
+  .media-grid__item {
+    position: relative;
+    background-color: var(--color-basic-black);
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
+
+.media-grid__image {
+  opacity: 1;
+  transition: opacity var(--animation-speed-default) ease-in-out;
+
+  [data-media-grid-variation='interactive'] .media-grid__item:hover & {
+    opacity: 0.8;
+  }
+}
+
+.media-grid__maximize {
+  position: absolute;
+  right: var(--size-spacing-4);
+  bottom: var(--size-spacing-4);
+  background: var(--color-basic-white);
+  height: 1.5rem;
```

Comment:
These height/width styles correspond with `var(--size-spacing-6)` and `var(--size-spacing-4)`. It makes sense to not apply them here as these variables aren't necessarily for dimensions. However, I think these would be better handled be a generic `var(--icon-size-height)` and `var(--icon-size-width)` or something more generic like `var(--icon-size-small)` - It looks like you've already implemented something similar that is specific for Alerts. Consider consolidating these styles: https://github.com/yalesites-org/component-library-twig/blob/develop/components/02-molecules/alert/_yds-alert.scss#L9

Reply:
These particular heights and widths are actually for the wrapping "circle" around the icon. But I did update the icon itself to follow that pattern.

It would be good to see if we can identify a set of sizes for icons across the board and reuse them... but I don't know if that's a thing. Either way, I've updated this implementation to be consistent with the one in alerts.

Thanks!

Reply:
ah yes you're right regarding the circle wrapper. Lets revisit the icon size in a follow up PR if this type of implementation needs to carry through to other components. This at least looks consistent with the alerts implementation.